pkg/kubelet/nodestatus/setters.go

```diff
@@ -364,19 +366,6 @@ func MachineInfo(nodeName string,
 			}
 			node.Status.Allocatable[k] = v
 		}
-		// for every huge page reservation, we need to remove it from allocatable memory
```

Comment:
This section of code is trying to remove the huge page resources from the allocatable memory. Your change is to factor out this to a separate function. Is my understanding correct that this is simply fixing a bug in this section of the code?

Reply:
remove it from allocatable memory when Updating Node Allocatable limit across pods